### ISSUE TITLE
Add Sharpen Image Module

### ISSUE THREAD

**Issue by vaibhavmatta** [publiclab/image-sequencer#1273]
### A Sharpen Module to make Image more sharper as per user's need should be added to Image-Sequencer.

> There is no module for Sharpening and Softening of the Images. This issue is for adding the Sharpen Module to Image-Sequencer.

Initial Image and Sharpened Images should look like,

Orignal Image:
![edgedetection1](https://user-images.githubusercontent.com/50797619/66708072-a27d1980-ed68-11e9-9409-761fdd880276.jpg)

Sharpened Image:
![edgedetection2](https://user-images.githubusercontent.com/50797619/66708074-aa3cbe00-ed68-11e9-9779-436d59e38a1f.jpg)

****

**Comment by vaibhavmatta** [comment]
@jywarren @HarshKhandeparkar I know how to do it and I am working on this module.
Thanks 😊

**Comment by HarshKhandeparkar** [comment]
Don't we have a noise reduction module? Are they similar?

**Comment by HarshKhandeparkar** [comment]
We appreciate your enthusiasm and your ideas are awesome. Just make sure to improve upon this module because the sharpened image shown in the issue body doesn't look very nice. It is not blurred but the artifacts....

**Comment by vaibhavmatta** [comment]
Thanks @HarshKhandeparkar
Noise reduction module is for reducing the Grains in the image. Sharpening module will work on edge detection algorithm and will increase the details of Image as per user's needs. 

I think our aim is not to make image look nice but to change the image According to the user. Yes, the Sharpened image in the issue body doesn't look nice but if user wants to Sharpen a colorful image with less edges details, this module will work awesomely for him.

**Comment by HarshKhandeparkar** [comment]
Ah ok now I understand properly. The image set me off initially. Thanks! You can ask @publiclab/is-reviewers for any help.

**Comment by vaibhavmatta** [comment]
@HarshKhandeparkar Yes.! I am working on it. I ll ask you and others if any help needed. Thanks!

**Comment by vaibhavmatta** [comment]
@jywarren What's your say? I think this module will add a new feature and that’s good for IS.
